### ISSUE TITLE
widen range of miragejs to allow the 0.2 alpha

### ISSUE THREAD

**Issue by NullVoxPopuli** [bgantzler/ember-mirage#42]
this library works with 0.2, but we get a peer error.

working with 0.2 is important, because it support MSW via https://github.com/miragejs/mirage-msw/tree/main

and MSW is a far superior debugging and logging experience than pretender.

**Comment by cah-brian-gantzler** [comment]
Currently it is ```"miragejs": "^0.1.47"```.  Should it be ```"miragejs": "^0.1.47 || ^0.2.0"``` or does alpha need to be in there somewhere?

**Comment by NullVoxPopuli** [comment]
we could try `>=` for 0.1.47?

**Comment by cah-brian-gantzler** [comment]
Figured there had to be an online checker
https://jubianchi.github.io/semver-check/#/%3E%3D%200.1.47/0.2.0-alpha.3

**Comment by NullVoxPopuli** [comment]
yay

**Comment by NullVoxPopuli** [comment]
Released in 0.3.2

**Comment by NullVoxPopuli** [comment]
oh something interesting -- npm is more strict than expected
```
npm error node_modules/miragejs
npm error   dev miragejs@"0.2.0-alpha.3" from the root project
npm error
npm error Could not resolve dependency:
npm error peer miragejs@">= 0.1.47" from ember-mirage@0.3.2
npm error node_modules/ember-mirage
npm error   dev ember-mirage@"0.3.2" from the root project
npm error
```
it doesn't consider  alphas to satisfy >=, contrary to that test link

**Comment by cah-brian-gantzler** [comment]
Did it pick up the change as the error is `peer miragejs@">= 0.1.47" from ember-mirage@0.3.2` I would have expected it to say `peer miragejs@">= 0.1.47 || >= 0.2.0-alpha.3" from ember-mirage@0.3.2` or it it ignoring the OR altogether?

**Comment by NullVoxPopuli** [comment]
it's now fixed with ember-mirage 0.3.3, where the peer is specified as >= 0.1 || >= 0.2.0-alpha.3
